.github/workflows/main.yml

```diff
@@ -1,47 +1,33 @@
 name: CI
+
 on: [push, pull_request]
+
 jobs:
   buildifier:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - name: Download Buildifier
+      - name: "Checkout the source code"
+        uses: actions/checkout@v1
+      - name: "Download Buildifier"
         run: |
           curl --location --fail https://github.com/bazelbuild/buildtools/releases/download/0.29.0/buildifier --output /tmp/buildifier
-          chmod +x /tmp/buildifier
-      - name: Lint Starlark Files
-        run: /tmp/buildifier -mode check -lint warn -r .
+          chmod +x /tmp/buildifier && echo "::add-path::/tmp/"
```

Comment:
This is weird but [not a hack](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/development-tools-for-github-actions#add-a-system-path-add-path).

Reply:
![image](https://user-images.githubusercontent.com/967132/70483565-c1b0d280-1a9d-11ea-86a7-410cac87ca48.png)